google/cloud/storage/batch.py

```diff
@@ -65,13 +64,8 @@ def __init__(self, method, uri, headers, body):
         lines.append("")
         lines.append(body)
         payload = "\r\n".join(lines)
-        if six.PY2:
-            # email.message.Message is an old-style class, so we
-            # cannot use 'super()'.
-            MIMEApplication.__init__(self, payload, "http", encode_noop)
-        else:  # pragma: NO COVER  Python3
-            super_init = super(MIMEApplicationHTTP, self).__init__
-            super_init(payload, "http", encode_noop)
+        super_init = super(MIMEApplicationHTTP, self).__init__
+        super_init(payload, "http", encode_noop)
```

Comment:
For Python3-only, `super()` is nice:
```suggestion
        super()__init__(payload, "http", encode_noop)
```